### PR TITLE
chore: add metadata to queue resources 

### PR DESCRIPTION
**Pull request by Lou1415926**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: internal/pkg/template/templates/workloads/partials/cf/subscribe.yml [hunk 1]
```diff
@@ -58,6 +58,8 @@ EventsKMSKey:
           Resource: '*'
```
Comment (efekarakus) [inline internal/pkg/template/templates/workloads/partials/cf/subscribe.yml:60]: Can we also add a description for the KMS key:
```
A KMS key to encrypt messages in your queues
```

File: internal/pkg/template/templates/workloads/partials/cf/subscribe.yml [hunk 2]
```diff
@@ -58,6 +58,8 @@ EventsKMSKey:
...
 
 EventsQueue:
+  Metadata:
+    'aws:copilot:description': 'An SQS Queue for queueing messages'
```
Comment (efekarakus) [inline internal/pkg/template/templates/workloads/partials/cf/subscribe.yml]:
```suggestion
    'aws:copilot:description': 'An events SQS queue to buffer messages'
```

File: internal/pkg/template/templates/workloads/partials/cf/subscribe.yml [hunk 3]
```diff
@@ -87,6 +89,8 @@ EventsQueue:
 
 {{- if .Subscribe.Queue}}{{- if .Subscribe.Queue.DeadLetter}}
 DeadLetterQueue:
+  Metadata:
+    'aws:copilot:description': 'An SQS Queue for queueing dead letters'
```
Comment (efekarakus) [inline internal/pkg/template/templates/workloads/partials/cf/subscribe.yml]:
```suggestion
    'aws:copilot:description': 'A dead letter SQS queue to buffer failed messages from the events queue'
```

File: internal/pkg/template/templates/workloads/partials/cf/subscribe.yml [hunk 4]
```diff
@@ -152,6 +156,8 @@ QueuePolicy:
 
 {{- if $topic.Queue}}
 {{logicalIDSafe $topic.Service}}{{logicalIDSafe $topic.Name}}EventsQueue:
+  Metadata:
+    'aws:copilot:description': 'An SQS Queue for queueing messages from the topic {{logicalIDSafe $topic.Name}}'
```
Comment (efekarakus) [inline internal/pkg/template/templates/workloads/partials/cf/subscribe.yml]: Nice!
```suggestion
    'aws:copilot:description': 'A SQS queue to buffer messages from the topic {{logicalIDSafe $topic.Name}}'
```

**Review by efekarakus — COMMENTED** [review]
Yay thank you! some suggestions for the messages 👇

File: internal/pkg/template/templates/workloads/partials/cf/subscribe.yml [hunk 5]
```diff
@@ -179,6 +185,8 @@ QueuePolicy:
 
 {{- if $topic.Queue.DeadLetter}}
 {{logicalIDSafe $topic.Service}}{{logicalIDSafe $topic.Name}}DeadLetterQueue:
+  Metadata:
+    'aws:copilot:description': 'An SQS Queue for queueing dead letters from the topic {{logicalIDSafe $topic.Name}}'
```
Comment (iamhopaul123) [inline internal/pkg/template/templates/workloads/partials/cf/subscribe.yml]:
```suggestion
    'aws:copilot:description': 'A dead letter SQS queue to buffer failed messages from the topic {{logicalIDSafe $topic.Name}}'
```

**Review by iamhopaul123 — COMMENTED** [review]
LGTM! Added a supplementary comment to Efe's feedback.

**Review by iamhopaul123 — APPROVED** [review]

**Review by efekarakus — APPROVED** [review]

**Pull request merged**